src/components/Calculator.js

```diff
@@ -0,0 +1,38 @@
+function Calculator() {
+  return (
+    <div className="calculator-container">
+      <input className="calc-input" type="text" placeholder="0" />
+      <div className="line">
+        <button type="button" className="key">AC</button>
+        <button type="button" className="key">+/-</button>
+        <button type="button" className="key">%</button>
+        <button type="button" className="key color">÷</button>
+      </div>
+      <div className="line">
+        <button type="button" className="key">7</button>
+        <button type="button" className="key">8</button>
+        <button type="button" className="key">9</button>
+        <button type="button" className="key color">x</button>
+      </div>
+      <div className="line">
+        <button type="button" className="key">4</button>
+        <button type="button" className="key">5/-</button>
+        <button type="button" className="key">6</button>
+        <button type="button" className="key color">-</button>
+      </div>
+      <div className="line">
+        <button type="button" className="key">1</button>
+        <button type="button" className="key">2</button>
+        <button type="button" className="key">3</button>
+        <button type="button" className="key color">+</button>
+      </div>
+      <div className="line">
+        <button type="button" className="key-exception">0</button>
+        <button type="button" className="key">.</button>
+        <button type="button" className="key color">=</button>
+      </div>
+    </div>
+  );
+}
+
+export default Calculator;
```

Comment:
Hi there, 
as seen your design needs to match the required design for this project 
your design ⬇️ 
![image](https://user-images.githubusercontent.com/60612329/211660688-e4c1a8bd-bf34-4e35-ac10-ec1b4ab94cd8.png)

the required design ⬇️ 
![image](https://user-images.githubusercontent.com/60612329/211660812-fa4f46f2-e46c-495e-a09d-3655c3dd8687.png)